cobbler/tftpgen.py

```diff
@@ -935,6 +935,48 @@ def build_kernel(
             autoinstall_meta, kernel_path, initrd_path, boot_loader
         )
 
+    def get_serial_kernel_options(self, append_line: str, arch: Archs, system = None):
+        """
+        Builds the console= kernel command line arguement
+
+        If system is none or serial options are not set, defaults are applied:
+        driver -> arch dependent
+        device -> 0
+        baud_rate -> 115200
+
+        If none of the serial options are set (may also happen for machine
+        unspecific boot menus), console= is only attached if:
+        always_enable_serial_console = true
+
+        Ideally all your servers stick to the defaults, then also unknown
```

Comment:
There are no sensible defaults.

Either specify console explicitly or rely on kernel detecting the right settings. Some firmware actually implements hints for the kernel where to find the console (and some does it wrong, too).